libtrap/src/ifc_tcpip.c

```diff
@@ -804,7 +802,7 @@ static int client_socket_connect(void *priv, const char *dest_addr, const char *
    int sockfd = -1, options;
    union tcpip_socket_addr addr;
    struct addrinfo *servinfo, *p = NULL;
-   int rv, addr_count = 0;
+   int rv = 0, addr_count = 0;
```

Comment:
`rv` can remain uninitialized because it is anyway rewritten on Line 821 (`getaddrinfo`) for `TCPIP` and it should be set according to my following comment for `UNIX` IFC.